### PR TITLE
Secure App Certificates

### PR DESCRIPTION
**Pull request by chiaramooney**
This PR is part of the effort to secure the certificate for RN Gallery. 

We now need to distinguish between the pipeline for PRs (original pipeline) and the new CI loop pipeline (see #83 for new CI Loop code). Pipelines which involve forks of the repo cannot have access to Azure DevOps Secure Files where the fresh pfx is now being stored. Thus the pipeline run for PRs cannot run tasks that depend on the certificate (i.e. building a release copy of the gallery requires a certificate). The new CI Loop will run daily Release builds of microsoft/react-native-gallery main and will publish the result. Since the CI Loop will run a build of the main repository, this pipeline will be able to access ADO Secure Files and thus will have access to the certificate to successfully build a Release copy of RN Gallery.

The changes in this PR are as following...

1. Removed the pfx file from the public repository since publicly publishing this data creates the risk of malicious code being signed with the "good" published certificate. 
2. Add .pfx to .gitignore so locally saved certificate will be ignored and not published.
3. Assign fresh certificate as the certificate for RN Gallery.
3. Cleaned up PR pipeline code to be only triggered for PRs. Removed the Release build of RN Gallery from PR pipeline.

To see the other changes connected to the securing certificate effort see #83.

### CODE REVIEW

**Comment by kmelmon** [comment]
Please have someone like Danny, Alex, or Nick double check these changes thanks!

**Review by kmelmon — APPROVED** [review]
:shipit:

**Comment by asklar** [comment]
I'm confused... the idea is to remove the PFX from the repo, not keep it and delete it in the CI build

**Comment by chiaramooney** [comment]
> I'm confused... the idea is to remove the PFX from the repo, not keep it and delete it in the CI build

Just updated the description for this PR and #83  to hopefully clarify! Changes across both these PRs paint the full picture of securing the app certificate.

File: windows/rngallery/Package.appxmanifest [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
  <Identity
     Name="48071f82-8a63-40b3-a728-2229057a8f83"
-    Publisher="CN=kmelmon"
+    Publisher="CN=chiaramooney"
```
Comment (asklar) [inline windows/rngallery/Package.appxmanifest:16]: probably need this (and the cert to be CN=Microsoft or Microsoft.ReactNative or something like that

**Review by asklar — APPROVED** [review]

File: windows/rngallery/Package.appxmanifest (hunk 1 above)
Comment (chiaramooney) [reply windows/rngallery/Package.appxmanifest:16]: Got it! Will be able to upgrade to a Microsoft Cert/Publisher once the app is in the store.

**Pull request merged**